ghcide/ghcide.cabal

```diff
@@ -272,25 +271,12 @@ library ghcide-test-utils
   visibility: public
   default-language:   GHC2021
 
-  hs-source-dirs: test/src test/cabal
+  hs-source-dirs: test/cabal
```

Comment:
It seems to me that we can now get rid of ghcide-test-utils entirely now that it consists of this one module?
https://github.com/haskell/haskell-language-server/blob/master/ghcide/test/cabal/Development/IDE/Test/Runfiles.hs

Reply:
cc @soulomoon ?

Reply:
Yes, test/src get merged with the hls-test-utils

Reply:
We will eventually drop all the uses of `locateGhcideExecutable`, after we are done with #4173